src/test/java/io/jenkins/docker/connector/DockerComputerConnectorTest.java

```diff
@@ -159,23 +165,41 @@ protected void should_connect_agent(DockerTemplate template) throws IOException,
         // FIXME on CI windows nodes don't have Docker4Windows
         Assume.assumeFalse(SystemUtils.IS_OS_WINDOWS);
```

Comment:
I tried removing this line but the tests failed saying "connection refused" to docker on localhost port 2375.
That implies that the Windows VMs we test with don't include docker.